adoc/admin-security-ldap.adoc

```diff
@@ -455,7 +455,8 @@ Example LDIF configuration to create group `k8s-admins` using an LDAP command:
   objectCategory: cn=Group,cn=Schema,cn=Configuration,dc=example,dc=org
 ====
 
-==== Example 2: Dex Active Directory TLS Connector Configuration (addons/dex/dex.yaml)
+==== Example 2: Dex Active Directory TLS Connector Configuration
```

Comment:
@cclhsu I've changed the Example 2 heading you had there because I didn't understand the content and how it relates to the example and what the steps are the user should take. Is the text below 'Dex connector template configured to use Active Directory:' what the user should add to the Dex ConfigMap?

Reply:
Dex connector can be configured to use different authentication service.  The example template is from Dex which provides user with detail descriptions on how to configure the Dex configmap using Active Directory.

Reply:
Ok, thanks, I've edited the bit below to make that a little clearer.